evadb/parser/evadb.lark

```diff
@@ -79,6 +79,12 @@ drop_table: DROP TABLE if_exists? uid
     
 drop_function: DROP FUNCTION if_exists? uid
     
+// SET statements (configuration)
+set_statement: SET config_name EQUAL_SYMBOL config_value
```

Comment:
If we want to be sync with Postgres, they support TO and =. Reference: https://www.postgresql.org/docs/7.0/sql-set.htm

Reply:
Done